src/Tickets/Seating/Frontend/Timer.php

```diff
@@ -480,7 +481,7 @@ public function ajax_interrupt(): void {
 
 		$post_type             = get_post_type( $post_id );
 		$post_type_object      = get_post_type_object( $post_type );
-		$has_tickets_available = tribe_tickets()->where( 'event', $post_id )->where( 'is_available', true )->count();
+		$has_tickets_available = $this->container->get( Frontend::class )->get_events_ticket_capacity_for_seating( $post_id );
```

Comment:
Why not set `Frontend` as a dependency in this class constructor? Does the instance of `Frontend` need to be fetched late?